operations/helm/charts/mimir-distributed/templates/lib/pdb.tpl

```diff
@@ -0,0 +1,26 @@
+{{/*
+Mimir common PodDisruptionBudget definition
+Params:
+  ctx = . context
+  component = name of the component
+  memberlist = true/false, whether component is part of memberlist
+*/}}
+{{- define "mimir.lib.podDisruptionBudget" -}}
+{{- $componentSection := include "mimir.componentSectionFromName" . }}
+{{ with (index $.ctx.Values $componentSection) }}
+{{- if .podDisruptionBudget -}}
+apiVersion: {{ include "mimir.podDisruptionBudget.apiVersion" $.ctx }}
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "mimir.resourceName" (dict "ctx" $.ctx "component" $.component) }}
+  labels:
+    {{- include "mimir.labels" (dict "ctx" $.ctx "component" $.component "memberlist" $.memberlist) | nindent 4 }}
```

Comment:
Should be be adding `app.kubernetes.io/part-of: memberlist` to a PodDisruptionBudget? IMO no. WDYT?

Reply:
yes, not much point. I think we're adding that label in some other places as well uselessly. Doesn't actually hurt as far as I know.